.github/workflows/test.yml

```diff
@@ -11,18 +11,24 @@ permissions:
   contents: read
 
 jobs:
-  integration:
-    name: Integration tests
+  build:
+    name: Build
     strategy:
       fail-fast: false
       matrix:
-        # macos-13-large is used as macos-13 seems too flaky.
-        # macos-14 (ARM) and later cannot be used for the most part of the job
-        # due to the lack of the support for nested virt.
-        #
-        # TODO: add macos-15-large https://github.com/lima-vm/socket_vmnet/pull/63
-        platform: [macos-13-large, macos-14-large]
+        platform: [macos-13, macos-14, macos-15]
```

Comment:
This build only on aarch64, previously we built only on x86_64. We don't wan to test build on both?

Reply:
Intel is covered in the integration test

Reply:
This will be easier to maintain if we use:

```yaml
platform:
- macos-13
- macos-14
- macos-15
```